sql/moz-fx-data-shared-prod/search/desktop_mobile_clients_daily_bizdev_report/view.sql

```diff
@@ -0,0 +1,34 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.search.desktop_mobile_clients_daily_bizdev_report`
+AS
+SELECT
+  client_id,
+  DATE_TRUNC(submission_date, MONTH) AS month,
```

Comment:
Should we keep `submission_date`, and use either `DATE_TRUNC` in SQL or the month option in `dimension_group` in Looker? https://docs.looker.com/reference/field-params/dimension_group#timeframe_options